MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -10,46 +11,58 @@
 namespace MauiAppBasic.MSALClient
 {
     /// <summary>
-    /// This is a wrapper for PCA. It is singleton and can be utilized by both application and the MAM callback
+    /// This is a wrapper for PublicClientApplication. It is singleton and can be utilized by both application and the MAM callback
     /// </summary>
     public class PCAWrapper
     {
         /// <summary>
-        /// This is the singleton used by consumers
+        /// This is the singleton used by consumers. Since PCAWrapper constructor does not have perf or memory issue, it is instantiated directly.

```

Comment:
Who's the consumer? Is this comment really saying anything? Can't we use public / private for hiding ?

Reply:
Changed consumers to Ux